emva1288/camera/camera.py

```diff
@@ -285,6 +309,15 @@ def grab(self, radiance, temperature=None, wavelength=None, f_number=None):
             u_y = self._u_e(radiance, wavelength=wavelength, f_number=f_number)
             img_e += np.random.poisson(u_y, size=self._shape)
 
+        # Bayer Filter simulation image
+        if self.bayer[0] is True:
+            pattern = self._pattern[0]
+            T_g, T_r, T_b = [self._transmition['green'],
+                             self._transmition['red'],
+                             self._transmition['blue']]
+            u_t = self._u_bayer(pattern, T_g, T_b, T_r)
+            img_e = np.multiply(img_e, u_t)
```

Comment:
why are you using `img_e`? the bayer process is before electrons creation, so the noise is not the right one